colour/recovery/jakob2019.py

```diff
@@ -470,8 +464,8 @@ def find_coefficients_Jakob2019(
     )
 
     def optimize(
-        target_o: ArrayLike, coefficients_0_o: ArrayLike
-    ) -> Tuple[NDArrayFloat, float]:
+        target_o: NDArrayFloat, coefficients_0_o: NDArrayFloat
+    ) -> Tuple[NDArrayFloat, float | np.float64]:
```

Comment:
Both `result.fun: np.float64` and `errors.error: float` are assignable to `float | np.float64`, and on `numpy >= 2.2` it's even equivalent to `float64` (`float64` has been made a proper subtype of `builtins.float`)